### PR TITLE
Call Event Types

### LINKED ISSUE #740: Add event type for calls for proposals
Add a new event type called "CALL" in the [EventType enum](https://github.com/deep-learning-indaba/Baobab/blob/0d68ea65d8f10eda05717e155aa5b104e45e1db0/api/app/events/models.py#L7)

Update Home.js to add a new section for these call event types

Switch off the registration stats in EventStats when the event type is call

### LINKED ISSUE #752: List past events that were attended
Request from EEML:
- Include past events that the user attended in the list of events on the organisation home page.

### PR DESCRIPTION
**Pull request by avishkar58**
Resolves #740 
Resolves #752

### CODE REVIEW

File: api/app/events/repository.py [hunk 1]
```diff
@@ -40,6 +43,17 @@ def get_upcoming_for_organisation(organisation_id):
                          .filter_by(organisation_id=organisation_id)\
                          .all()
 
+    @staticmethod
+    def get_attended_by_user_for_organisation(organisation_id, user_id):
+        # .filter_by(candidate_response=True)\
```
Comment (amritpurshotam) [inline api/app/events/repository.py:48]: Some stray commented code

File: api/app/events/tests.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def seed_static_data(self):
         db.session.add(self.test_form)
         db.session.commit()
 
-        db.session.flush()
+        # db.session.flush()
```
Comment (amritpurshotam) [inline api/app/events/tests.py:50]: Did this fix some error? Can remove if so?

**Review by amritpurshotam — APPROVED** [review]
At least the back end stuff looks good!
Just some stray commented code

File: api/app/events/tests.py (hunk 2 above)
Comment (avishkar58) [reply api/app/events/tests.py]: Yeah, it was for the dreaded detached instance error :)

**Comment by avishkar58** [comment]
Thanks for the review @amritpurshotam !

**Pull request merged**